charts/nfs-server-provisioner/templates/priorityclass.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.priorityClass.create -}}
+kind: PriorityClass
+apiVersion: scheduling.k8s.io/v1
+metadata:
+  name: {{ .Values.priorityClass.name }}
```

Comment:
```suggestion
  name: {{ or .Values.priorityClass.name (include "nfs-provisioner.fullname" .) }}
```

Reply:
Since this won't ever be a global default PriorityClass resource, it seems a bit cumbersome to require the user to specify a name explicitly for this.